app/controllers/dish_ingredients_controller.rb

```diff
@@ -0,0 +1,10 @@
+class DishIngredientsController < ApplicationController
```

Comment:
Nice controller organization! This is a great RESTful solution to this joins resource CRUD operation